samples/hello_everybody/hello_everybody.c

```diff
@@ -80,6 +80,7 @@ static int init_hello(void)
 		pr_err("Failed to create dir: %ld", PTR_ERR(dir));
 		return PTR_ERR(dir);
 	}
+	/* easiest way would be smthing like - debugfs_create_u64(..., (u64 *)&jiffies); */
 	struct dentry *jiff_file = debugfs_create_file("jiffies", 0444, dir, NULL, &jiffies_fops);
 
 	if (IS_ERR(jiff_file)) {
```

Comment:
btw these checks are correct, but it seems that error checking is not necessary for debugfs create functions anymore - https://git.kernel.org/linus/8bcbde2bb1374112264826aa26ff959149f318e4

Reply:
These checks may be a bit useless, but the task was to work with debugfs, no to use it for logging or something minor, so I thought it would be a good idea to check if my solution doesn't fail.

Reply:
About `debugfs_create_u64(..., (u64 *)&jiffies)`:
jiffies is a `unsigned long volatile` variable, so I [believe](https://en.cppreference.com/w/c/language/volatile) the pointer to it must not be passed as a non-volatile one.